src/config.ts

```diff
@@ -6,10 +6,17 @@ const config = t.type({
   BUCKET_NAME: t.string,
   EXTERNAL_ID: t.string,
   ROLES_TO_ASSUME: t.string,
+  UPLOAD_DELAY: t.number,
 });
+
+const parsedEnv = {
+  ...process.env,
+  UPLOAD_DELAY: parseInt(process.env.UPLOAD_DELAY, 10),
```

Comment:
`setTimeout` takes a delay of milliseconds. Is the default intended to be 10ms here or should it be 10s? (lacking domain knowledge here).
Also to avoid confusion we could call the env `UPLOAD_DELAY_SECONDS` or `..._MILLIS` to avoid misconfiguration.

Reply:
[`parseInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#radix). JS whackiness :sweat_smile:

Reply:
Ahh, of course. 🤦‍♂️ 
My JS days are fortunately long gone so I forgot about that. 😅